cli/config/config.go

```diff
@@ -48,25 +47,25 @@ type ImageDefinition struct {
 
 // Project represents the project configuration.
 type Project struct {
-	APIVersion   string            `yaml:"apiVersion"`
-	Kind         string            `yaml:"kind"`
-	Metadata     Metadata          `yaml:"metadata"`
-	Capabilities []Capabilities    `yaml:"capabilities"`
-	Suites       []Suite           `yaml:"suites"`
-	Files        []string          `yaml:"files"`
-	Image        ImageDefinition   `yaml:"image"`
-	BeforeExec   []string          `yaml:"beforeExec"`
-	Timeout      int               `yaml:"timeout"`
-	Sauce        SauceConfig       `yaml:"sauce"`
-	Env          map[string]string `yaml:"env"`
-	Parallel     bool              `yaml:"parallel"`
+	APIVersion string            `yaml:"apiVersion"`
+	Kind       string            `yaml:"kind"`
+	Metadata   Metadata          `yaml:"metadata"`
+	Settings   []Settings        `yaml:"settings"`
```

Comment:
I was actually suggesting to remove Capabilities/Settings support from the `project`, but retaining backwards compatibility in the `suite`.

Capabilities weren't heavily used by users back when we supported them the project level.
There is no point in keeping it at this point, now that we recommend the use of suites.

Reply:
OK. I see